modules/Bio/EnsEMBL/Variation/Pipeline/Remapping/PreRunChecks.pm

```diff
@@ -50,7 +50,6 @@ sub run {
 
   # we need at least bioperl version 1.006924
   my $bioperl_version = Bio::Perl->VERSION;  
-  $self->warning($bioperl_version);
   die "At least Bio::Perl 1.006924 required" if ($bioperl_version < 1.006924);
```

Comment:
Does BioPerl version need to be checked in runnable that uses it to check for changes since pipeline pre checks and doing the analysis